### PR TITLE
Veeam Batch Fix + Add replication advanced metrics button

### PR DESCRIPTION
**Pull request by ChengYanJin**
Bump UserEvent to v14 and jest to v27

Reshape the Veeam Assistant Configuration Form:
image

Add advanced metrics in the Workflow
image

### CODE REVIEW

File: src/react/ui-elements/Veeam/VeeamSummary.tsx [hunk 1]
```diff
@@ -188,7 +188,7 @@ export const VeeamSummary = ({
           content={
             <WrapperWithWidth>
               <HideCredential credentials={secretKey} />
-              <Clipboard text={accessKey} />
+              <Clipboard text={secretKey} />
```
Comment (JBWatenbergScality) [inline src/react/ui-elements/Veeam/VeeamSummary.tsx]: Shall we add a test for the copy buttons ?

Comment (ChengYanJin) [reply src/react/ui-elements/Veeam/VeeamSummary.tsx]: Upgrade to UserEvent v14 by the way as well !!!!

**Review by JBWatenbergScality — APPROVED** [review]
LGTM !!!!!
bnha-anime-gif

**Pull request merged**